packaging/debian/shopify-cli/DEBIAN/postinst

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
+
+gem install shopify-cli
```

Comment:
👏 
This is really interesting! How does it work for updating the gem though?

Reply:
If the Debian package version is bumped, the developer would be able to install the new version (via apt upgrade or install or any other means). That would re-run this post-install script which would then trigger an update of the gem - unless I misunderstand how `gem install` works for version changes.

One thing we could consider is whether it's worth having some sort of `package` rake task that updates the control file with the current ruby version (and possibly forcing the version of the gem to install?).

Reply:
`gem install` won't do anything if a version of gem is already present, I think you'll need `gem update`.

Reply:
~`gem install` won't do anything if a version of the gem is already present, I think you'll need `gem update`.~

Nope, I was wrong, `gem install` does seem to update.

Reply:
Ok, that's good to know. If that wasn't the case, we could of course always call something like `install; update` to make sure it does both.